backend/src/services/model.ts

```diff
@@ -235,10 +248,8 @@ export async function updateModelCard(
   metadata: unknown,
 ): Promise<ModelCardRevisionDoc> {
   const model = await getModelById(user, modelId)
-
-  const auth = await authorisation.model(user, model, ModelAction.Update)
-  if (!auth.success) {
-    throw Forbidden(auth.info, { userDn: user.dn, modelId })
+  if (model.settings.mirror.sourceModelId !== '' || model.settings.mirror.destinationModelId !== '') {
```

Comment:
This check can be the same are the other ones- if the source ID has been set, don't allow it. No need to take destination ID into account